src/full/Agda/Interaction/Options/Base.hs

```diff
@@ -563,6 +565,7 @@ infectiveCoinfectiveOptions =
   , infectiveOption (isJust . optCubical) "--cubical/--erased-cubical"
   , infectiveOption optGuarded "--guarded"
   , infectiveOption optProp "--prop"
+  , infectiveOption optLevelUniv "--level-universe"
```

Comment:
Shouldn't this option be *coinfective* rather than infective? I.e. you should be able to import a module that uses `--level-universe` from one that does not, but not the other way around.

Reply:
I agree with you, it makes more sense for it to be co-infective. Making it so implies making the primitive/builtin modules use `--level-universe` as well.

Reply:
Yes, that makes sense to me.